web_app/dist/main.css

```diff
@@ -0,0 +1,117 @@
+html {
```

Comment:
Discussed offline, but I'm not a fan of styling the `html` element. This might be useful https://css-tricks.com/html-vs-body-in-css/